Update binary_sensor.august for Smart Lock DoorSense

ehendrix23 opened the pull request:
**Description:**
Added text that the August component will also create a binary sensor for the DoorSense sensor of the 3rd Gen Smart Lock Pro.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17299

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

ehendrix23 committed:
Update binary_sensor.august for Smart Lock DoorSense

Added text that the August component will also create a binary sensor for the DoorSense sensor of the 3rd Gen Smart Lock Pro.

frenck commented on source/_components/binary_sensor.august.markdown:
```diff
@@ -2,14 +2,14 @@
-ha_category: Doorbell
-ha_release: "0.64"
+ha_category: Doorbell, Door
+ha_release: "0.80"
```
Please do not update the `ha_release` in this case, it is used to indicate the first version where the component was introduced to Home Assistant. This is just an addition to an existing component.

ehendrix23 replied:
Apologies, changed back to 0.64

frenck commented on source/_components/binary_sensor.august.markdown:
```diff
@@ -2,14 +2,14 @@
 title: "August Binary Sensor"
 description: "Instructions on how to integrate your August devices into Home Assistant."
-date: 2018-02-17 22:00
+date: 2018-10-03 15:00
```
Please do not update the date for this.

ehendrix23 replied:
Changed back as well. :-)

frenck commented on source/_components/binary_sensor.august.markdown:
```diff
@@ -21,3 +21,7 @@ If you have August Doorbell, once you have enabled the [August component](/compo
 * Doorbell motion sensor
 * Doorbell online sensor
 
+If you have August Smart Lock with DoorSense, once you have enabled the [August component](/components/august/), you should see following sensor:
```
`you should see the following sensors:`

ehendrix23 replied:
I'm assuming here to insert the word "the"? Just did that.

If not, not sure what is meant then.

frenck commented on source/_components/binary_sensor.august.markdown:
```diff
@@ -2,14 +2,14 @@
 logo: august.png
-ha_category: Doorbell
-ha_release: "0.64"
+ha_category: Doorbell, Door
```
Please only specify only 1 category. Secondly, `Door` isn't a valid category.

ehendrix23 replied:
OK, it was doorbell but really now it would be for both doorbell and door sensor. Hence why I added "door". :-)

I changed it to Binary Sensor instead as I think that is really where it probably falls under. If incorrect, can you please let me know where I can find the list of valid components?

Appreciate it!

frenck reviewed (changes requested):
@ehendrix23 I've reviewed your PR and left some comments. Could you please take a look? 👍

ehendrix23 committed:
Updated based on feedback provided

Put date entry back to original date.
Put ha_release back to original release.
Changed ha_category from "Doorbell, Door" to "Binary Sensor". Originally it was "Doorbell".
Added the word "the" in the sentence "you should see the following sensors:" 
Added the letter "s" to the word "sensor" in the sentence "you should see the following sensors:"

ehendrix23 commented:
@frenck Reviewed and updated. Thanks for helping me. Please let me know if there is anything still missing or requiring to be updated.

Thank you!!!

ehendrix23 commented:
@frenck Currently it had DoorBell there as category. The thing with calling it Lock category is that this is a binary sensor for August. When someone has a August doorbell it will create 3 sensors for that.
I added that when someone has a lock with the sensor it will then also create a binary sensor for the lock.
Thus calling it for "Lock" would I think be wrong for the doorbell sensors. DoorBell as it was called would be wrong for the door sensor from the door lock.

frenck commented:
I feels weird... Nevertheless, I fail to categorize it appropriately as well.
So I approve it for now. Can be merged as soon as the parent PR gets merged.

ehendrix23 commented:
Updated pull request from home-assistant/home-assistant#17116 to home-assistant/home-assistant#17299.

frenck committed:
Merge branch 'next' into patch-1
